src/get_estimation.py

```diff
@@ -3,31 +3,61 @@
 
 
 import time
-from .benchmark_mediation import *
-from rpy2.rinterface_lib.embedded import RRuntimeError
 import sys
+from rpy2.rinterface_lib.embedded import RRuntimeError
 import pandas as pd
 import numpy as np
+from .benchmark_mediation import *
 
 
 def get_estimation(x, t, m, y, estimator, config):
+    """Wrapper estimator fonction ; calls an estimator given mediation data
+    in order to estimate total, direct, and indirect effects.
+
+    Parameters
+    ----------
+    x : array-like, shape (n_samples, n_features_covariates)
+        Covariates value for each unit
+    t : array-like, shape (n_samples)
+        Treatment value for each unit
+    m : array-like, shape (n_samples, n_features_mediator)
+        Mediator value for each unit
+    y : array-like, shape (n_samples)
+        Outcome value for each unit
+    estimator : str
```

Comment:
thanks for the docstring
for the estimator, the full list should be included in the docstring (maybe with more details about each estimator)